tools/chefdk.sh

```diff
@@ -16,10 +16,13 @@
 # limitations under the License.
 #
 
+# Fail fast
+set -x
```

Comment:
If you want to fail fast, you'll want to use `set -e`. `-x` simply prints out the commands. https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

Reply:
@tduffield yup, you're right. Thanks for that catch and sorry for the super late update. 😅